### DISCUSSION TITLE
Add config option to disable rope

### DISCUSSION THREAD

**Discussion by rsaihe** [vectorwing/FarmersDelight#375] [General]
Currently there is no way to disable the rope from this mod. A config option might be helpful to pack developers.

**Comment by vectorwing** [comment]
You can disable ropes (and any crafted item) by overriding its crafting recipe with a datapack.

Adding configs to disable every core item of a mod would get excessive very quick, and determining what to open such a config for is even more complicated. I'd rather keep configs for more relevant, non-datapackable changes.

**Reply by Aceplante** [reply]
Only ropes, not the foods which is why everyone downloaded this pack in the first place

**Comment by rsaihe** [comment]
Alright, that makes sense. Thank you!
